src/app.constants.ts

```diff
@@ -13,27 +13,27 @@ export const PATH_DATA: Record<
   YOUR_SERVICES: { url: "/your-services" },
   ENTER_PASSWORD: { url: "/enter-password" },
   ADD_MFA_METHOD: {
-    url: "/add-mfa-method",
+    url: "/choose-backup",
     event: EventType.SelectedApp,
     type: UserJourney.AddMfaMethod,
   },
   ADD_MFA_METHOD_APP: {
-    url: "/add-mfa-method-app",
+    url: "/choose-backup-app",
     event: EventType.ValueUpdated,
     type: UserJourney.AddMfaMethod,
   },
   ADD_MFA_METHOD_APP_CONFIRMATION: {
-    url: "/add-mfa-method-app-confirmation",
+    url: "/choose-backup-app-confirmation",
     event: EventType.Confirmation,
     type: UserJourney.AddMfaMethod,
   },
   ADD_MFA_METHOD_SMS: {
-    url: "/add-mfa-method-sms",
+    url: "/choose-backup-sms",
     event: EventType.VerifyCodeSent,
     type: UserJourney.AddMfaMethod,
   },
   ADD_MFA_METHOD_SMS_CONFIRMATION: {
-    url: "/add-mfa-method-sms-confirmation",
+    url: "/choose-backup-sms-confirmation",
```

Comment:
I don't think any URLs should be changing as part of this ticket, except for the `choose-backup` one.